### DISCUSSION TITLE
[Theme] Hourly forecast first

### DISCUSSION THREAD

**Discussion by zxbmmmmmmmmm** [zxbmmmmmmmmm/SkylineWeather#62] [Themes]
![image](https://github.com/user-attachments/assets/a02b5e6c-d254-4aa0-aac1-f600bf77228a)
**1.3.x**:
[HourlyFirst.zip](https://github.com/user-attachments/files/16470953/HourlyFirst.zip)

**Comment by RickyVercetti** [comment]
Hi, I traslate all app in spanish in crowdin.
I love this app
I hope to help

**Comment by RickyVercetti** [comment]
By the way, the hourly forecast first look better than now, bue I think they need more space between them
Cheers
